lib/unleash/strategies.rb

```diff
@@ -23,7 +23,13 @@ def fetch(name)
     end
 
     def add(strategy)
-      @strategies[strategy.name] = strategy
+      if default_strategy_names.include?(strategy.name)
+        # rubocop:disable Style/StderrPuts
```

Comment:
I really, really don't want people to disable this and miss it but I'm not sure what impacts this has on loggers

Reply:
What kind of impact are you thinking about?

Reply:
Logger not reading it correctly really, moot point though, moved to a log error now